core/jazz_ui/e2e/07-slsappservices/slsappservices.e2e-spec.ts

```diff
@@ -0,0 +1,451 @@
+/*
+ *Copyright 2016-2017 T Mobile, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); You may not use
+ * this file except in compliance with the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser, element, by, protractor, $ } from 'protractor';
+import { Jazz } from '../page-objects/jazzservices.po';
+import { Timeouts, Browser } from 'selenium-webdriver';
+import { SSL_OP_SSLEAY_080_CLIENT_DH_BUG, SSL_OP_TLS_BLOCK_PADDING_BUG } from 'constants';
+import { Common } from '../common/commontest';
+
+
+xdescribe('Overview', () => {
```

Comment:
putting an x here will make the entire sls-app test suite not run. please remove it and in other places

Reply:
done in new commit